src/FSharp.Core/array.fsi

```diff
@@ -797,7 +797,7 @@ module Array =
     ///
     /// (inputs1, inputs2) ||> Array.exists2 (fun a b -> a > b)
     /// </code>
-    /// Evaluates to <c>false</c>
+    /// Throws <c>ArgumentException</c>
```

Comment:
Second example will throw as well. I think we should instead make one example which returns true, one which returns false and one which throws.